device/mellanox/x86_64-nvidia_sn5640-r0/Mellanox-SN5640-C512S2/hwsku.json

```diff
@@ -0,0 +1,202 @@
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet248": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet256": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet264": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet272": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet280": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet288": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet296": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet304": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet312": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet320": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet328": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet336": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet344": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet352": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet360": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet368": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet376": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet384": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet392": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet400": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet408": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet416": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet424": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet432": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet440": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet448": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet456": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet464": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet472": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet480": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet488": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet496": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet504": {
+            "default_brkout_mode": "8x100G[50G]"
+        },
+        "Ethernet512": {
```

Comment:
Need to check how to make sure the speed for service port is 10GB - maybe need to add configuration in port_config.ini

Reply:
Update - port_config.ini won't override hwsku.json configurations. 
We have a way to do it by changing hwsku.json like in Mellanox-SN4700-O8C48 SKU, checking if this way is valid.

Reply:
We can do it using hwsku.json, because we have this commit - https://github.com/sonic-net/sonic-buildimage/pull/9278
Done.